backend/ops_api/features/portfolio/research_projects/list_research_projects_funding.feature

```diff
@@ -0,0 +1,64 @@
+Feature: List Research Projects and their funding for a given Portfolio and FY.
+  (Displayed on the "Projects and Spending" tab on the Portfolio details
+  page in the Research Projects table.)
+
+  N.B. The term "Managed CAN" below indicates that the CAN is managed by the given Portfolio.
+  The assumption here is that the Portfolio Details page should only include CANs that it manages.
+  Relationship is: 1 Managed CAN has only 1 Portfolio it belongs to.
+
+  Scenario: There exist Research Projects that have managed CANs that are managed by a Portfolio.
+    Given a set of data below with the current FY 2023
+      | Portfolio | Managed RP | FY   | Managed CAN |
+      | 1         | 1          | 2023 | 1,2,3       |
+      | 1         | 2          | 2023 | 1,2         |
+      | 1         | 3          | 2023 | 1           |
+      | 1         | 4          | 2023 | 1           |
+
+    When I list the Research Projects for a given Portfolio and FY 2023
+    Then the result should be the 4 Research Projects above.
+
+  Scenario: Calculate Research Project total funding for a given FY.
+    Given a set of data below with the current FY 2023
+      | Research Project | Managed CAN | CAN FY | Funding |
+      | 1                | 1           | 2023   | $5      |
+      | 1                | 2           | 2023   | $7      |
+      | 1                | 3           | 2023   | $3      |
+      | 1                | 1           | 2022   | $5      |
+      | 1                | 2           | 2022   | $5      |
+      | 2                | 4           | 2023   | $5      |
+
+    When I calculate the total funding for a Research Project in FY 2023
+    Then the result should be $15.
+
+  Scenario: Calculate Research Project funding to date.
```

Comment:
I'd like to understand more why RP1 would be $10 and 2 would be $5.  What is the current date in this scenario ?